testsuite/Main.hs

```diff
@@ -618,6 +618,7 @@ runClashTest = defaultMain $ clashTestRoot
         , runTest "VecOfSum" def{hdlSim=False}
         , runTest "T452" def{hdlSim=False}
         , runTest "T895" def{hdlSim=False,hdlTargets=[VHDL]}
+        , runTest "T1360" def{hdlSim=False, hdlTargets=[VHDL], clashFlags=["-fclash-hdlsyn", "Vivado"]}
```

Comment:
Should this be tested with other flags than `-fclash-hdlsyn Vivado`? I know the issue was only mentioning Vivado, but should this hold for `Quartus` or `Other`?